src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs

```diff
@@ -110,6 +115,11 @@ public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, s
                     queryStringParams.Add("format", format);
                 }
 
+                if (appendVersion?.Equals("true", StringComparison.OrdinalIgnoreCase) == true)
```

Comment:
```suggestion
                if (appendVersion?.Equals("true", StringComparison.OrdinalIgnoreCase))
```

Reply:
You might need to check if you need `?? false`

Reply:
`appendVersion?.Equals("true", StringComparison.OrdinalIgnoreCase) == true`
and
`appendVersion?.Equals("true", StringComparison.OrdinalIgnoreCase) ?? false`
yields the same result in every case I think.

But if you prefer `appendVersion?.Equals("true", StringComparison.OrdinalIgnoreCase) ?? false` I can change it.